pom.xml

```diff
@@ -324,8 +331,8 @@
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.277.x</artifactId>
-        <version>984.vb5eaac999a7e</version>
+        <artifactId>bom-weekly</artifactId>
```

Comment:
I think we prefer depending on a specific line of bom, rather than weekly, unless this is some 'global' approach change that I'm not aware of?

Reply:
But there is no LTS release available yet with the Guava upgrade, so there is no BOM line available yet with the Guava upgrade. The only Jenkins release with the Guava upgrade is a weekly, and the correct BOM `artifactId` for pre-LTS weekly releases is `bom-weekly`. I suggest you read the documentation in `jenkinsci/bom`.